docs/reference/content/getting-started/quick-tour-primer.md

```diff
@@ -0,0 +1,93 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Quick Tour Primer"
+[menu.main]
+  parent = "Getting Started"
+  identifier = "Primer"
+  weight = 20
+  pre = "<i class='fa'></i>"
++++
+
+# Quick Tour Primer
+
+The aim of this guide is to provide background about the Scala driver and its asynchronous API before going onto 
+looking at how to use the driver and MongoDB.
+
+{{% note %}}
+See the [installation guide]({{< relref "getting-started/installation-guide.md" >}})
+for instructions on how to install the MongoDB Scala Driver.
+{{% /note %}}
+
+## The Observable API
+
+One of the goals of the Scala driver is not force users to take on any specific library dependencies which may conflict with the users chosen libraries. To achieve this the Scala API makes use of an custom implementation of the Observer pattern which comprises three traits:
+
+1. Observable
+2. Observer
+3. Subscription 
+
+An `Observable` is a provider of a potentially unbounded number of sequenced elements, published according to the demand received from it's `Subscription`.
```

Comment:
And then below can we say something about the ease of creating implicit adapters from the custom API to third-party libraries?